src/main/scala/scalismo/ui/model/capabilities/Transformable.scala

```diff
@@ -52,6 +54,15 @@ trait Transformable[T] extends RenderableSceneNode with Grouped {
     publishEvent(Transformable.event.GeometryChanged(this))
   }
 
+  var p: SceneNode = parent
```

Comment:
Would it make more sense if this would be moved into a function which is then called? ( init() or setupListener() ) Otherwise, I would argue that the var should be private at least.